backend/src/datastore/PostDataStore.js

```diff
@@ -0,0 +1,59 @@
+const {DataSource} = require('apollo-datasource');
+
+class PostDataStore extends DataSource {
+
+    constructor(userDataStore, posts = null) {
+        super()
+
+        this.userDataStore = userDataStore;
+        this.posts = posts ?? [
+            {
+                id: 1,
+                title: 'Message 1',
+                votes: [],
+                author_id: 'Max Mustermann'
+            },
+            {
+                id: 2,
+                title: 'Message 2',
+                votes: [
+                    'Max Mustermann'
+                ],
+                author_id: 'Max Mustermann2'
+            }
+
+        ];
```

Comment:
I would use a dedicated `seed(postDataSource)` function and initialize the datasource with empty array here.